packages/widget-chat/src/components/activity-item-system-message/index.js

```diff
@@ -16,27 +18,45 @@ export default function ActivityItemSystemMessage(props) {
   } = props;
 
   let systemMessage;
-  let displayName;
-  let actionVerb;
-  if (isSelf) {
-    displayName = `You`;
-    actionVerb = `your`;
-  }
-  else {
-    displayName = name;
-    actionVerb = `their own`;
-  }
 
   if (verb === SYSTEM_MESSAGE_VERB_CREATE) {
-    systemMessage = `${displayName} created conversation. ${timestamp}`;
+    if (isSelf) {
+      systemMessage = ( // eslint-disable-line no-extra-parens
+        <FormattedMessage
+          defaultMessage={`You created conversation.`}
```

Comment:
This was the only string I did not take from the web client. The only options were:
```
someoneCreatedThisRoom: 'This starts the "{roomTitle}" room, created by {name}.',
youCreatedThisRoom: 'This starts the "{roomTitle}" room.',
```